ide/ui/ide-monaco/src/main/resources/META-INF/dirigible/ide-monaco/js/editor.js

```diff
@@ -605,6 +611,54 @@ function getModuleImports(text) {
     return moduleImports;
 }
 
+function setupPrettier(fileName) {
+    let editorApi = "/services/v4/ide/editor/prettier/config";
+    let prettierConfigRequest = `${editorApi}${fileName}`
+
+    fetch(prettierConfigRequest, {
+        method: "GET"
+    })
+    .then(function(response) { return response.json(); })
+    .then(function(configJson) {
+        registerPrettierFormatter(configJson, "javascript");
+    })
+    .catch((error) => {
+        let defaultConfigJson = {
+              "trailingComma": "es5",
```

Comment:
This is not a JSON, this is a JS object. Please rename the variable accordingly and remove the quotes of the object properties' names.